pkgs/development/python-modules/unstructured/default.nix

```diff
@@ -99,31 +123,133 @@ buildPythonPackage {
   src = fetchFromGitHub {
     owner = "Unstructured-IO";
     repo = "unstructured";
-    tag = version;
+    rev = "refs/tags/${version}";
     hash = "sha256-+I5eXG/ICmYPDTavDnyLlopIvoABjdDwOyfotrNs6qs=";
   };
 
   propagatedBuildInputs = [
+    # Base dependencies
+    anyio
+    backoff
+    beautifulsoup4
+    certifi
+    cffi
     chardet
+    charset-normalizer
+    click
+    cryptography
+    dataclasses-json
+    deepdiff
+    emoji
     filetype
+    h11
+    html5lib
+    httpcore
+    httpx
+    idna
+    joblib
+    # jsonpath-python
+    langdetect
     lxml
-    msg-parser
+    marshmallow
+    mypy-extensions
+    nest-asyncio
     nltk
-    openpyxl
-    pandas
-    pdf2image
-    pdfminer-six
-    pillow
-    pypandoc
-    python-docx
-    python-pptx
+    numpy
+    olefile
+    orderly-set
+    packaging
+    psutil
+    pycparser
+    pypdf
+    python-dateutil
+    # python-iso639
     python-magic
-    markdown
+    # python-oxmsg
+    rapidfuzz
+    regex
     requests
-    tabulate
-    xlrd
+    requests-toolbelt
+    six
+    sniffio
+    soupsieve
+    tqdm
+    typing-extensions
+    typing-inspect
+    # unstructured-client
+    urllib3
+    webencodings
+    wrapt
   ];
 
+  optional-dependencies = rec {
+    all-docs = csv ++ docx ++ epub ++ pdf ++ req-markdown ++ odt ++ org ++ pptx ++ xlsx;
```

Comment:
How about naming it `all` instead?

Reply:
all-docs is what they used in their setup.py file.
If you feel strongly about the name though, im happy to change

Reply:
Fair enough, let's leave it like that then.